lib/mauth/dice_bag/mauth.yml.dice

```diff
@@ -5,6 +5,8 @@ common: &common
   mauth_api_version: v1
   app_uuid: <%= configured.mauth_app_uuid! || 'fb17460e-9868-11e1-8399-0090f5ccb4d3' %>
   private_key_file: config/mauth_key
+  authenticate_with_only_v2: <%= configured.authenticate_with_only_v2 || 'false' %>
```

Comment:
The effect and usecase of these configuration options probably need to be explained in the README or other doc.
At least would need to address when should I turn this on. Or if should I wait till the client updates the template?